docs/deployment/docker.md

```diff
@@ -169,11 +170,11 @@ RUN find packages \! -name "package.json" -mindepth 2 -maxdepth 2 -exec rm -rf {
 # Stage 2 - Install dependencies and build packages
 FROM node:16-bullseye-slim AS build
 
-# install sqlite3 dependencies
+# Install sqlite3 and isolated-vm dependencies, if you don't use sqlite you can remove libsqlite3-dev
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
     apt-get update && \
-    apt-get install -y --no-install-recommends libsqlite3-dev python3 build-essential && \
+    apt-get install -y --no-install-recommends libsqlite3-dev python3 g++ build-essential && \
```

Comment:
@awanlin sqlite3 as apt-get package installation are lost between Dockerfile stages (here at the end of this build stage). You can remove libsqlite3-dev from here, else you are right, python3 g++ build-essential are needed to "yarn install" at line 187/188

Reply:
Sorry, I made and update after you left this comment and I'm not sure what line this is on or if maybe I already have this updated correctly. Can you take a look, please? It can wait till tomorrow

Reply:
just turning this line to
```
apt-get install -y --no-install-recommends python3 g++ build-essential
```

will be enough

Reply:
Perfection